Gopkg.toml

```diff
@@ -41,10 +37,6 @@ required = [
   name = "github.com/cenkalti/backoff"
   version = "1.1.0"
 
-[[constraint]]
-  branch = "master"
-  name = "github.com/docker/distribution"
```

Comment:
`docker/distribution` and `Azure/go-autorest` weren't imported or required. I don't know why we have them here.
W/o them `dep ensure` could complete in reasonable time for me.

Reply:
Weird, dep would complain if they are not used. Anyway :+1: